src/components/AddList.jsx

```diff
@@ -9,11 +9,12 @@ export default function AddList({ setListPath }) {
 	const navigate = useNavigate();
 	const { user } = useAuth();
 
-	const handleSubmit = (event) => {
-		event.preventDefault();
+	const handleSubmit = async (e) => {
+		e.preventDefault();
```

Comment:
Nice attention to detail here by changing "event.preventDefault()" to "e.preventDefault" for the sake of consistency. The rest of our application uses "e" and not "event" to represent "event". Good call.